server/manifest.go

```diff
@@ -7,5 +7,5 @@ var manifest = struct {
 	Version string
 }{
 	ID:      "com.mattermost.welcomebot",
-	Version: "1.2.0",
```

Comment:
@jwilander we should probably leave the version out of this PR, and then once merged kick off the "next release" playbook